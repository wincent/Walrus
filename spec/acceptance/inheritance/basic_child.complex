#extends BasicParent
#block footer
This is the overridden footer.
#end