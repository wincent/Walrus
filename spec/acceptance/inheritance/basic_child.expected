This is the base template which subclasses will inherit.

This is the overridden footer.
